src/excalidraw-app/collab/CollabWrapper.tsx

```diff
@@ -427,6 +445,52 @@ class CollabWrapper extends PureComponent<Props, CollabState> {
     this.excalidrawAPI.history.clear();
   };
 
+  private initializeIdleDetector = () => {
+    let intervalId: number | null = null;
+    let timeoutId: number | null = null;
```

Comment:
I'd rename these to `idleTimeoutId` and `activeIntervalId` so it's clear.

Reply:
Done!